portal/src/graphql/adminapi/mutations/generateOOBOTPCodeMutation.graphql

```diff
@@ -0,0 +1,5 @@
+mutation generateOOBOTPCodeMutation($target: String!) {
+  generateOOBOTPCode(input: { target: $target }) {
+    code
+  }
+}
```

Comment:
We do not need to add this. We are going to expose this feature to the portal for the moment.